content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_storages.rb

```diff
@@ -21,6 +21,7 @@ def main
                     values_hash[nil] = 'None'
                   else
                     provider.storages.each do |storage|
```

Comment:
We have a `#writable_storages` which checks the read_only bit on the host/storage mount.  Do non-data storage domains set read-only?  Could be a nicer way to skip thse.

Reply:
Note that `#writable_storages` is only available from the `Host` model today.  That method is already exposed to automate.

Reply:
Ah you're right for some reason I thought we had that exposed at the provider level, good catch